pkgs/by-name/lu/lumafly/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  zlib,
+  icu,
+  fontconfig,
+  openssl,
+  xorg,
+  icoutils,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+buildDotnetModule rec {
+  pname = "lumafly";
+  version = "3.2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "TheMulhima";
+    repo = pname;
```

```suggestion
    repo = "lumafly";
```

https://github.com/NixOS/nixpkgs/issues/277994